src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/DrawOrderCallbackHandler.java

```diff
@@ -127,6 +127,7 @@ private CallbackResponse copyDraftToCaseDocument(CallbackParams callbackParams)
         CCDCollectionElement<CCDClaimDocument> claimDocument = CCDCollectionElement.<CCDClaimDocument>builder()
             .value(CCDClaimDocument.builder()
                 .documentLink(draftOrderDoc)
+                .documentName(draftOrderDoc.getDocumentFileName())
```

Comment:
can we add size here as well to be consistent with other documents?

Reply:
size is not available to add, can look on it later